IPython/nbconvert/utils/lexers.py

```diff
@@ -49,10 +49,14 @@
 line_re = re.compile('.*?\n')
 
 ipython_tokens = [
-  (r'(\%+)(\w+)\s+(\.*)(\n)', bygroups(Operator, Keyword,
+  (r'(%)(sx|sc|system)(.*)(\n)', bygroups(Operator, Keyword,
                                        using(BashLexer), Text)),
-  (r'(\%+)(\w+)\b', bygroups(Operator, Keyword)),
```

Comment:
`(%%?)` would be fine, there's no need to handle more than two %. But I think we do need to handle it here:

1. These lexers are not only used in highlighting notebooks. Despite being in nbconvert, this module is imported by `IPython.sphinxext.ipython_console_highlighting`, which is used for IPython syntax in Sphinx docs.
2. I don't think the code you linked to catches all cell magics, only a subset that have a highlighting language specifically associated with them: https://github.com/ipython/ipython/blob/3.0.0-dev/IPython/nbconvert/preprocessors/highlightmagics.py#L39